modules/mobile-agents/android/client/plugins/IDPProxy/src/org/wso2/mdm/agent/proxy/IdentityProxy.java

```diff
@@ -81,17 +94,17 @@ public void init(String clientID, String clientSecret, String username, String p
 		editor.commit();
 		setAccessTokenURL(tokenEndPoint);
 		AccessTokenHandler accessTokenHandler =
-		                                        new AccessTokenHandler(clientID, clientSecret,
-		                                                               username, password,
-		                                                               tokenEndPoint, this);
+				new AccessTokenHandler(clientID, clientSecret,
+				                       username, password,
+				                       tokenEndPoint, this);
 		accessTokenHandler.obtainAccessToken();
 	}
 
 	public void getToken(Context contextt, TokenCallBack call, String clientID, String clientSecret)
-	                                                                                                throws Exception,
-	                                                                                                InterruptedException,
-	                                                                                                ExecutionException,
-	                                                                                                TimeoutException {
+			throws Exception,
```

Comment:
Exception handling in this method needs to be fixed. There are quite a few issues that need to be fixed such as avoiding throwing "Exception", properly handling "InterruptedException", etc.